tools/generate-sponsors.ts

```diff
@@ -190,7 +190,7 @@ async function main(): Promise<void> {
       allSponsorsConfig,
       (_, value: unknown) => value ?? undefined,
       2,
-    ),
+    ) + '\n',
```

Comment:
Heh, alternately we could import from the `prettier` package (https://prettier.io/docs/en/api.html#prettierformatsource-options)...

Reply:
> but I think there are some cases where the former will add new lines that prettier wouldn't.

Yes, with arrays.

Reply:
priettier is actually smart with json, it preserves new lines with objects, thus there is no issue here

Reply:
Yeah this is my point - prettier will potentially reformat the JSON.

i.e. for the json blob `{"a":[1,2,3,4,5,6],"b":[1,2,3,4,5]}`,
`JSON.stringify(x, null, 2)` results in
```json
{
  "a": [
    1,
    2,
    3,
    4,
    5,
    6
  ],
  "b": [
    1,
    2,
    3,
    4,
    5
  ]
}
```
but `prettier.format` for that stringified output results in
```json
{
  "a": [1, 2, 3, 4, 5, 6],
  "b": [1, 2, 3, 4, 5]
}
```

Which is why I say that if we're adding the newline for prettier, then we should enforce the blob is just formatted with prettier at generation time to ensure it's correct!

Reply:
ok, I applied `prettier` as a formatter